From 776d6e31762e6a2c62da05c939587e0e44feddbb Mon Sep 17 00:00:00 2001
From: Maxime Jourdan <maxi.jourdan@wanadoo.fr>
Date: Thu, 5 Apr 2018 22:13:53 +0200
Subject: [PATCH] dts: Actually set VDEC_1 clock to 666MHz

Thanks narmstrong for the tip.
Should allow to test H.264 decoding at full speed.

Also drop the VDEC_HEVC out of here since it wasn't used by the pwrc anyway,
and the pwrc itself is going away in the future..
---
 arch/arm64/boot/dts/amlogic/meson-gxl.dtsi | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxl.dtsi b/arch/arm64/boot/dts/amlogic/meson-gxl.dtsi
index ff2ba1087c5d2..f868ba69576b5 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxl.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-gxl.dtsi
@@ -766,8 +766,10 @@
 	clocks = <&clkc CLKID_VPU_INTR>, <&clkc CLKID_DOS_PARSER>, <&clkc CLKID_VDEC_1>, <&clkc CLKID_VDEC_HEVC>;
 	clock-names = "vpu_intr", "dos_parser", "vdec_1", "vdec_hevc";
 
-	assigned-clocks = <&clkc CLKID_VDEC_1>,
-			  <&clkc CLKID_VDEC_HEVC>;
-	assigned-clock-rates = <648000000>,
-			       <648000000>;
+	assigned-clocks = <&clkc CLKID_VDEC_1_SEL>,
+			  <&clkc CLKID_VDEC_1>;
+	assigned-clock-parents = <&clkc CLKID_FCLK_DIV3>,
+				 <0>;
+	assigned-clock-rates = <0>,
+			       <666666666>;
 };
